bindings/csharp/http/batch.cs

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
nit: Should we have the file name as "Program.cs" to match our other examples? Application everywhere!

Reply:
Good catch, @amulyavarote.  I would not in this case because it's more in the style of the C# 6 minimal API to use a single program.cs.